### PR TITLE
Add Stable mgmt for Confidential Ledger

### PR DESCRIPTION
**Pull request by musabbir**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [✔️ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [✔️ ] Other, please clarify - Stable swagger for GA. Same as last approved preview version. 
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [✔️ ] SDK of .NET (need service team to ensure code readiness)
      - [✔️ ] SDK of Python
      - [✔️ ] SDK of Java
      - [✔️ ] SDK of Js
      - [ ✔️] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [✔️ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [✔️ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [✔️ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [✔️ ] Check this box if any of the following appy to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [✔️ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/confidentialledger/resource-manager/readme.md [hunk 1]
```diff
@@ -69,11 +69,14 @@ This is not used by Autorest itself.
...
 swagger-to-sdk:
   - repo: azure-sdk-for-net
+  - repo: azure-sdk-for-net-track2
   - repo: azure-sdk-for-python-track2
   - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-java-track2
   - repo: azure-sdk-for-go
   - repo: azure-sdk-for-go-track2
   - repo: azure-sdk-for-js
+  - repo: azure-sdk-for-js-track2
```
Comment (qiaozha) [inline specification/confidentialledger/resource-manager/readme.md:77]: don't need this, the azure-sdk-for-js is using track2 generator.

File: specification/confidentialledger/resource-manager/readme.md [hunk 2]
```diff
@@ -69,11 +69,14 @@ This is not used by Autorest itself.
 ```yaml $(swagger-to-sdk)
 swagger-to-sdk:
   - repo: azure-sdk-for-net
+  - repo: azure-sdk-for-net-track2
```
Comment (qiaozha) [inline specification/confidentialledger/resource-manager/readme.md:71]: @ArcturusZhang Does .net SDK track2 use this way ?

File: specification/confidentialledger/resource-manager/readme.md [hunk 3]
```diff
@@ -69,11 +69,14 @@ This is not used by Autorest itself.
...
   - repo: azure-sdk-for-python-track2
   - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-java-track2
```
Comment (qiaozha) [inline specification/confidentialledger/resource-manager/readme.md:74]: @weidongxu-microsoft does jave track2 use this way ?

File: specification/confidentialledger/resource-manager/readme.typescript.md [hunk 4]
```diff
@@ -6,8 +6,8 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 ```yaml $(typescript)
 typescript:
   azure-arm: true
-  package-name: "confidentialledger"
-  output-folder: "$(typescript-sdks-folder)/packages/confidentialledger"
+  package-name: "mgmt-confidentialledger"
```
Comment (qiaozha) [inline specification/confidentialledger/resource-manager/readme.typescript.md:9]: please use @azure/arm-confidentialledger as the package name. the @azure/arm- prefix is for all mgmt RP.

File: specification/confidentialledger/resource-manager/readme.typescript.md [hunk 5]
```diff
@@ -6,8 +6,8 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
...
+  output-folder: "$(typescript-sdks-folder)/packages/mgmt-confidentialledger"
```
Comment (qiaozha) [inline specification/confidentialledger/resource-manager/readme.typescript.md:10]: $(typescript-sdks-folder)/sdk/confidentialledger/arm-confidentialledger

File: specification/confidentialledger/resource-manager/readme.csharp.md [hunk 6]
```diff
@@ -10,6 +10,6 @@ csharp:
   payload-flattening-threshold: 1
   clear-output-folder: true
   client-side-validation: false
-  namespace: Microsoft.ConfidentialLedger
-  output-folder: $(csharp-sdks-folder)/confidentialledger/management/Microsoft.ConfidentialLedger/GeneratedProtocol
+  namespace: Azure.Management.ConfidentialLedger
+  output-folder: $(csharp-sdks-folder)/confidentialledger/Azure.Management.ConfidentialLedger/GeneratedProtocol
```
Comment (qiaozha) [inline specification/confidentialledger/resource-manager/readme.csharp.md:14]: @ArcturusZhang Does this change make sense for csharp ?

**Review by qiaozha — COMMENTED** [review]
@msyyc can you take a look at the Python breaking change ? Thanks

File: specification/confidentialledger/resource-manager/readme.md (hunk 1 above)
Comment (musabbir) [reply specification/confidentialledger/resource-manager/readme.md:77]: Done

File: specification/confidentialledger/resource-manager/readme.csharp.md (hunk 6 above)
Comment (musabbir) [reply specification/confidentialledger/resource-manager/readme.csharp.md]: Updated to Azure.ResourceManager.ConfidentialLedger based on this page.
https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespaces-approved-list

Please let me know if this is more appropriate

**Comment by msyyc** [comment]
approved for python breaking change

**Comment by ArcturusZhang** [comment]
@fengzhou-msft could you please help about the details on azure-sdk-for-net-track2 pipelines? Some details are not looking correct

**Comment by fengzhou-msft** [comment]
> @fengzhou-msft could you please help about the details on azure-sdk-for-net-track2 pipelines? Some details are not looking correct

Confidential Ledger only has a data plane track 2 package, therefore this PR has no impact on azure-sdk-for-net-track2.

File: specification/confidentialledger/resource-manager/readme.md (hunk 3 above)
Comment (weidongxu-microsoft) [reply specification/confidentialledger/resource-manager/readme.md]: Musabbir is updated it and track2 removed.

This is correct for Java.

**Review by weidongxu-microsoft — APPROVED** [review]
LGTM for Java

**Comment by musabbir** [comment]
@akning-ms Could you help merge this PR?

**Review by ArcturusZhang — APPROVED** [review]
Approve for Go SDK related configurations

**Comment by musabbir** [comment]
> Hi @musabbir, can you confirm what you did for this PR just move preview version 2021-5-13-preview? if any other change in swagger in new version, we may need ARM to sign off, and BTW, is there new version deployed to production? if no, when it will be deployed?

There are no changes other than copying over the last approved swagger (2021-5-13) to a stable folder.

This version is fully supported in production.

**Comment by musabbir** [comment]
> > Hi @musabbir, can you confirm what you did for this PR just move preview version 2021-5-13-preview? if any other change in swagger in new version, we may need ARM to sign off, and BTW, is there new version deployed to production? if no, when it will be deployed?
> 
> There are no changes other than copying over the last approved swagger (2021-5-13) to a stable folder.
> 
> This version is fully supported in production.

We will update the ARM Manifest after this PR is merged.

**Pull request merged**